//#include<iostream>
//using namespace std;
//
//void fun() {
//	throw "exception has occured";
//}
//
//int main() {
//	try {
//		fun();
//	}
//	catch (const char* e) {
//		cout << e << endl;
//	}
//
//	return 0;
//}